doc/releases/changelog-0.23.0.md

```diff
@@ -140,59 +195,118 @@
   `method='pyscf'` in :func:`~.molecular_hamiltonian`.
```

Comment:
woah I had no idea we could use `:func:` inside the changelog (using markdown) and it would work!!!!

Reply:
We should do this more to create links to docstrings, _but_ it does mean we have to strip this out before copying to the github release notes :(